js/src/framework.ts

```diff
@@ -15,15 +15,15 @@ import pluralize from "pluralize";
 
 export type Metadata = Record<string, unknown>;
 
-export interface EvalCase<Input, Output> {
+export interface EvalCase<Input, Output, Expected> {
```

Comment:
Do we need the `Output` template param here if it isn't even part of the type? Should it only need to be part of `EvalScorerArgs`?

Reply:
we do not. fixed